pandas/tests/reshape/test_concat.py

```diff
@@ -1279,6 +1279,33 @@ def test_concat_ignore_index(self, sort):
 
         tm.assert_frame_equal(v1, expected)
 
+    concat_index_names = [
```

Comment:
I feel like we would want to put this inside `pytest.mark.parametrize` rather than defining a separate variable

Reply:
all integrated now